tools/minishift-build/atlasmap.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-. "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/vars.sh"
+. "$(cd "$(dirname "$(readlink -f "$0")")" && pwd)/vars.sh"
```

Comment:
Sorry, but `-f` is not supported on OS X. Could we revert it to the original version ?